rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -39,6 +39,12 @@ then
       chkconfig rudder-agent on
     fi
   fi
+else
+  # Delete all flag files
+  rm -f /var/rudder/tmp/migration-rudder-service-rename
+  rm -f /var/rudder/tmp/migration-rudder-service-systemd
+  rm -f /var/rudder/tmp/migration-rudder-service-enabled
+  rm -f /var/rudder/tmp/migration-rudder-cf-serverd-disabled
```

Comment:
Why do you do this here ? They are used just afterwards

Reply:
Yes, but if the `rudder-server-root` is installed after `rudder-agent`, we need to keep the flags (as the init file will be gone too)

Reply:
how does removing them at the begining of the postinst prevent that ?

Reply:
It will absolutely not, you're right, do you have an idea to handle this problem?